integration_tests/README.md

```diff
@@ -0,0 +1,56 @@
+# Integration Tests
+
+## Running Tests
+
+To run integration tests:
+
+`make integration`
+
+## Making Tests
+
+### 1. Fixtures
+
+Fixtures Directory: `integration_tests/fixtures`
+
+- Folders in the fixtures directory will be built as docker images in alphabetical order
+- Each folder should contain a `Dockerfile` and any resources necessary to build the image
+- The resulting image will be named `cbfix_fixture_name`
+
+*Note*: Since fixtures are created in alpha order, they can have FROM directives for previously created images
+
+### 2. Tests
+
+Tests Directory: `integration_tests/tests`
+
+- Folders in the tests directory will be run as tests
```

Comment:
Actually, we don't technically need Golang on the host environment.  the compose script can bring up a golang container to do testing instead of doing it locally.

Really I added the go language to `.travis.yml` because,  by default, TravisCI uses RVM and Ruby.  It just makes it clear which language we are targeting - has no actual effect the way we are using it.